cmd/server.go

```diff
@@ -109,7 +112,56 @@ func produceServerCmdRunFunc(o serverOperator, action int) cmdRunFunc {
 				log.Fatalf("Removing Server failed: %s", err)
 			}
 		}
+	case serverCreateAction:
+		return func(cmd *cobra.Command, args []string) {
+			ctx := context.Background()
+			cServer, err := client.CreateServer(ctx, gsclient.ServerCreateRequest{
+				Name:   servernameFlag,
+				Cores:  cpuFlag,
+				Memory: memFlag,
+			})
+			if err != nil {
+				log.Fatal("Create server has failed with error", err)
+			}
+			log.WithFields(log.Fields{
+				"server_uuid": cServer.ObjectUUID,
+			}).Info("Server successfully created")
 
+			if templateFlag != "" {
+				template, _ := client.GetTemplateByName(ctx, templateFlag)
+				cStorage, err := client.CreateStorage(ctx, gsclient.StorageCreateRequest{
+					Name:        string(servernameFlag),
+					Capacity:    storageFlag,
+					StorageType: gsclient.DefaultStorageType,
+					Template: &gsclient.StorageTemplate{
+						TemplateUUID: template.Properties.ObjectUUID,
+						Password:     passwordFlag,
+						PasswordType: gsclient.PlainPasswordType,
+						Hostname:     hostnameFlag,
+					},
+					Labels: []string{"label"},
```

Comment:
Don't give an arbitrary label here.

Reply:
> Looks okay in general.
> Please follow https://chris.beams.io/posts/git-commit/#seven-rules. Can you force push to redo the commit message please?

#### Yes, i will do as far all changes are made properly.

Reply:
> Don't give an arbitrary label here.
Should I completely remove it for now or possibly catch it with a flag?